design/common.blocks/link/_theme/link_theme_islands.styl

```diff
@@ -18,6 +18,21 @@
         {
             color: #070;
         }
+
+        &_ghost
+        {
+            color: #999;
+        }
+
+        &_black
+        {
+            color: #000;
+        }
+
+        &_strong
```

Comment:
Where do this names (`ghost`, `black`, `strong`) come from? They don't look too semantic.